File: backend/KvantasServer/KvantasServer/Models/Invoice.cs
namespace KvantasServer.Models
{
    public class Invoice
    {
        public string BuyerName { get; set; }

        public string PhoneNumber { get; set; }
        
        public string Address { get; set; }
        
        public string ProductName { get; set; }

        public int Amount { get; set; }

        public int ProductPrice { get; set; }

        public override string ToString()
        {
            return $"{{ {nameof(BuyerName)}: \"{BuyerName}\"," +
                $"{nameof(PhoneNumber)}: \"{PhoneNumber}\"," +
                $"{nameof(Address)}: \"{Address}\"," +
                $"{nameof(ProductName)}: \"{ProductName}\"," +
                $"{nameof(Amount)}: \"{Amount}\"," +
                $"{nameof(ProductPrice)}: \"{ProductPrice}\"" +
                $"}}";
        }
    }
}

File: frontend/nbp-projekat2/src/app/models/Invoice.ts
export class Invoice{

    buyerName: string;
    phoneNumber: string;
    address: string;
    productName: string;
    amount: number;
    productPrice: number;

    constructor(buyerName: string, phoneNumber: string, address: string, productName: string, amount: number, productPrice: number ){
        this.buyerName = buyerName;
        this.phoneNumber = phoneNumber;
        this.address= address;
        this.productName = productName;
        this.amount = amount;
        this.productPrice = productPrice;
    }
}